app/page.tsx

```diff
@@ -64,7 +64,7 @@ export default function Home() {
       </div>
       <div className="flex flex-col pt-3 pb-3 mt-12 overflow-auto">
         <LoadingComponent isLoading={loading} values={usdValues}>
-          <div className="flex gap-3">{usdValues && pills.map((pill) => <Pill key={pill.id} pill={pill} data={usdValues[pill.token]} />)}</div>
+          <div className="gap-3">{usdValues && pills.map((pill) => <Pill key={pill.id} pill={pill} data={usdValues[pill.token]} />)}</div>
```

Comment:
think the overflow here is a feature not a bug. 
<img width="1289" alt="image" src="https://github.com/terra-money/alliance-dashboard/assets/3447315/5879a827-eecb-426a-8e6a-b570bf254432">
looks slightly worse now